src/javashopper.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import os
+import time
+
+import keyboard
+import mouse
+import screen
+
+from config import Config
+from logger import Logger
+from npc_manager import NpcManager, Npc
+from template_finder import TemplateFinder, load_template
+
+
+def exit():
+    Logger.info("Exiting...")
+    os._exit(0)
+
+
+class JavaShopper:
+    """
+    Shop javazon 20 ias 3 java skill gloves
+    """
+
+    def __init__(self, config):
+        self._screen = screen.Screen()
+        self.config = config
+        self._template_finder = TemplateFinder(self._screen)
+        self._npc_manager = NpcManager(
+            screen=self._screen, template_finder=self._template_finder
+        )
+        keyboard.add_hotkey(self.config.general["exit_key"], exit)
+
+    def run(self):
+        Logger.info("STARTING JAVAZON GG GLOVES SHOPPER!")
+
+        # Are we at the right start place?
+        t1 = time.time()
+        while True:
+            success, pos = self._template_finder.search_and_wait("A5_RED_PORTAL")
+            if success:
+                mouse.move(*pos, duration=0.1)
+                mouse.click()
+                break
+
+            if time.time() > t1 + 10:
+                Logger.info("Cannot find start place :( exiting...")
+                os._exit(0)
+            time.sleep(3)
+
+        self.shop_loop()
+
+    def shop_loop(self):
+        while True:
+            time.sleep(3.6)
+            success, pos = self._template_finder.search_and_wait("A5_RED_PORTAL")
+            mouse.move(*pos, duration=0.1)
+            mouse.click()
+            time.sleep(1)
+            self._npc_manager.open_npc_menu(Npc.ANYA)
+            self._npc_manager.press_npc_btn(Npc.ANYA, "trade")
+            time.sleep(0.1)
+            img = self._screen.grab()
+
+            # 20 IAS gloves have a unique color so we can skip all others
+            ias_glove_found, pos = self._template_finder.search(
+                ref=load_template("assets/templates/javashopper/ias_gloves.png", 1.0),
+                inp_img=img,
+                threshold=0.98,
+                roi=self.config.ui_roi["vendor_stash"],
+            )
+            if ias_glove_found:
+                mouse.move(*pos, duration=0.1)
```

Comment:
`search()` will return screen/img coordinates. If someone has a two monitor setup this might be offset by one screen width.
You can convert it with `screen.convert_screen_to_monitor()`

Same for all the other template search coordinates.

Reply:
Ah I see.

I added `normalize_monitor` kwarg to search() method now. Let me know if you think that's a goo idea.

Reply:
I like it!